ranger/core/actions.py

```diff
@@ -974,6 +974,9 @@ def display_help(self):
             if process.poll() != 16:
                 return
         process = self.run(['man', 'ranger'])
+        if process is None:
+            self.notify("Show manpage failed, check if man is installed", bad=True)
```

Comment:
Please rephrase the message to emphasize the failure "Failed to show man page, check if man is installed".

Reply:
OK, updated.